### PR TITLE
Improve docker image generation

### PR DESCRIPTION
**Pull request by phiz71**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/improve-docker-image-generation/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-rest-api/docker/Dockerfile-dev [hunk 1]
```diff
@@ -20,12 +20,11 @@ LABEL maintainer="contact@graviteesource.com"
 ARG GRAVITEEIO_VERSION=0
 ENV GRAVITEEIO_HOME /opt/graviteeio-management-api
 
-ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.zip .
+ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.tar.gz /tmp/
```
Comment (gaetanmaisse) [inline gravitee-apim-rest-api/docker/Dockerfile-dev:23]: I think we can directly put it at the right place:
```suggestion
ADD ./gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION}.tar.gz ${GRAVITEEIO_HOME}
```

And remove:
```
     && mv /tmp/gravitee-apim-rest-api-distribution-${GRAVITEEIO_VERSION} ${GRAVITEEIO_HOME} \
```

File: gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/gravitee-apim-gateway-standalone-distribution-zip/src/main/assembly/plugin-assembly-distribution.xml [hunk 2]
```diff
@@ -23,8 +23,7 @@
     <id>bin</id>
 
     <formats>
-        <format>zip</format>
-	<format>tar.gz</format>
+	    <format>tar.gz</format>
```
Comment (gaetanmaisse) [inline gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/gravitee-apim-gateway-standalone-distribution-zip/src/main/assembly/plugin-assembly-distribution.xml:26]: Should we not keep the zip to send them to `artifactory?`/`download.io`?/??

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/improve-docker-image-generation/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/gravitee-apim-gateway-standalone-distribution-zip/src/main/assembly/plugin-assembly-distribution.xml (hunk 2 above)
Comment (phiz71) [reply gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/gravitee-apim-gateway-standalone-distribution-zip/src/main/assembly/plugin-assembly-distribution.xml:26]: the distribution zip (with policies) is not send to download.gravitee.io nor on artifactory (for the moment).

File: gravitee-apim-rest-api/docker/Dockerfile-dev (hunk 1 above)
Comment (phiz71) [reply gravitee-apim-rest-api/docker/Dockerfile-dev:23]: I tried, but then i was not able to start my docker container, because a file was missing

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**